node/src/inputs.jsx

```diff
@@ -0,0 +1,96 @@
+import * as Fluent from '@fluentui/react';
+import { ButtonAdapter, InputAdapter } from '@/shiny.react';
+
+export const ActionButton = ButtonAdapter(Fluent.ActionButton);
+export const CommandBarButton = ButtonAdapter(Fluent.CommandBarButton);
+export const CommandButton = ButtonAdapter(Fluent.CommandButton);
+export const CompoundButton = ButtonAdapter(Fluent.CompoundButton);
+export const DefaultButton = ButtonAdapter(Fluent.DefaultButton);
+export const IconButton = ButtonAdapter(Fluent.IconButton);
+export const PrimaryButton = ButtonAdapter(Fluent.PrimaryButton);
+
+export const Calendar = InputAdapter(Fluent.Calendar, (value, setValue) => ({
```

Comment:
so this is created manually, is that correct? but we're still using some of the work that the generator did, like documentation?

Reply:
Correct.